src/server/sample_data/data_sets/ecommerce/index.js

```diff
@@ -0,0 +1,191 @@
+        type: 'integer'
+      },
+      customer_last_name: {
+        type: 'keyword'
+      },
+      customer_phone: {
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+          }
+        }
+      },
+      day_of_week: {
+        type: 'keyword'
+      },
+      day_of_week_i: {
+        type: 'integer'
+      },
+      email: {
+        type: 'keyword'
+      },
+      manufacturer: {
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+          }
+        }
+      },
+      order_date: {
+        type: 'date'
+      },
+      order_id: {
+        type: 'integer'
+      },
+      products: {
+        type: 'nested',
+        properties: {
+          base_price: { type: 'float' },
+          discount_percentage: { type: 'float' },
+          quantity: { type: 'integer' },
+          manufacturer: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+              }
+            }
+          },
+          tax_amount: { type: 'float' },
+          product_id: { type: 'integer' },
+          category: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+              }
+            }
+          },
+          sku: { type: 'keyword' },
+          taxless_price: { type: 'float' },
+          unit_discount_amount: { type: 'float' },
+          min_price: { type: 'float' },
+          _id: { type: 'keyword' },
+          discount_amount: { type: 'float' },
+          created_on: { type: 'date' },
+          product_name: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+              }
+            }
+          },
+          price: { type: 'float' },
+          taxful_price: { type: 'float' },
+          base_unit_price: { type: 'float' },
+        }
+      },
+      sku: {
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+          }
+        }
+      },
+      taxful_total_price: {
+        type: 'float'
+      },
+      taxless_total_price: {
+        type: 'float'
+      },
+      total_quantity: {
+        type: 'integer'
+      },
+      total_unique_products: {
+        type: 'integer'
+      },
+      type: {
+        type: 'keyword'
+      },
+      user: {
+        type: 'keyword'
+      },
+      geoip: {
+        type: 'nested',
+        properties: {
+          country_iso_code: { type: 'keyword' },
+          location: { type: 'geo_point' },
+          region_name: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+              }
+            }
+          },
+          continent_name: { type: 'keyword' },
+          city_name: { type: 'keyword' }
+        }
+      }
+    },
+    timeFields: ['order_date'],
+    currentTimeMarker: '2018-01-09T00:00:00',
```

Comment:
`currentTimeMarker ` is causing the time issue you are seeing. `currentTimeMarker` is the reference to now in your data set.  When the data is installed, timestamps are translated based on the present time.  The distance between a timestamp and `currentTimeMarker` is used to translate the data set. For the eCommerce data, it looks like the values occur during December 2016. The reason you where seeing the big gap between the current date and the translated data is that your value of `2018-01-09T00:00:00` translated the data with a gap of 13 months.

<img width="2043" alt="screen shot 2018-09-17 at 5 47 13 am" src="https://user-images.githubusercontent.com/373691/45621489-b9561d00-ba3d-11e8-947d-ca9fda007a7b.png">

Setting this value to `2016-12-11T00:00:00` seems like a reasonable value that translates the data appropriately.

<img width="2036" alt="screen shot 2018-09-17 at 5 51 38 am" src="https://user-images.githubusercontent.com/373691/45621682-6761c700-ba3e-11e8-9813-faef48bcaf99.png">

Reply:
ahh, thanks Nathan! Makes sense, oversight on my end.